sea-query-binders/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "sea-query-binders"
+version = "0.1.0"
+authors = [ "Valentin Tolmer <valentin@tolmer.fr>" ]
+edition = "2021"
+description = "Binder traits for connecting sea-query with SQL engines"
+license = "MIT OR Apache-2.0"
+documentation = "https://docs.rs/sea-query"
+repository = "https://github.com/SeaQL/sea-query"
+categories = [ "database" ]
+keywords = [ "database", "sql", "mysql", "postgres", "sqlite" ]
+
+[lib]
+
+[dependencies]
+sea-query = { version = "^0.24", path = "../" }
+sqlx = { version = "^0.5.6", optional = true }
+
+[features]
```

Comment:
Please, add support for: `with-ipnetwork` and `with-mac_address`

Reply:
Done.